src/mongocxx/test/spec/unified_tests/operations.cpp

```diff
@@ -0,0 +1,989 @@
+// Copyright 2020 MongoDB Inc.
```

Comment:
The actual logic to run these operations was copied from `operation.cpp` with a few tweaks. The newer logic is for special commands (e.g., `assertDifferentLsidOnLastTwoCommands`) and commands we didn't implement in the past (e.g., `upload`). The `run` function at the bottom of this class does all of the switchings.